docs/segment.md

```diff
@@ -2,98 +2,188 @@
 
 !!! warning
-    It is advisable to ensure the quality of the segmentation before changing the order of the lines because adding or deleting lines systematically restarts the calculation of this order, overwriting manual modifications in the process.
\ No newline at end of file
+    If you created text regions to annotate text lines, you will usually want your text lines to be linked to them. To do so, select your baselines, and click on the **link selected lines to the background region**, or press ++y++. If you want to unlink them, just press the button again, or press ++u++.
+
+!!! note
+    It is possible to automatically perform [text lines segmentation](predict.md#predict-the-segmentation) with a machine learning model. 
+
+### Reordering text lines
+
+The reading order of the text lines is calculated automatically. You can display the order number of each text lines from the “Segmentation” panel by clicking on the **Toggle ordering display** button or by pressing ++l++.
+
+You can also go to the “Text” panel (also accessible with the shortcut ++ctrl+5++) where the lines are displayed in the reading order.
+
+In this panel, and after the text lines are transcribed, it is possible to modify the reading order by clicking “Toggle sorting mode.” A simple drag and drop is enough to carry out the modifications.
+
+![image: Line reordering](img/segment/line_order.gif "You can reorder text lines in eScriptorium.")
+
+!!! warning
+    Please make sure the quality of the segmentation is good enough before changing the order of the lines because adding or deleting lines restarts the line ordering, overwriting manual modifications in the process.
+
+### Advanced text lines segmentation features
+
+- You can add coordinate points on a text line by double-clicking  at the desired location on the baseline.
+
+![image: Adding new coordinates points to a baseline.](img/segment/adding_point_baseline.gif "You can add coordinate points on a text line by double-clicking  at the desired location on the baseline.")
+
+- You can move a coordinate point with a simple click and drag.
+
+![image: Moving coordinates point on a baseline.](img/segment/move_point_baseline.gif "You can move a coordinate point with a simple click and drag.")
+
+- You can delete a coordinate point on a baseline by clicking on it, then click on the thrash button or press ++ctrl+delete++.
+
+![image: Deleting coordinates point on a baseline.](img/segment/delete_point_baseline.gif "You can delete a coordinate point on a baseline by clicking on it, then click on the thrash button or press ++ctrl+delete++.")
+
+- You can cut one or several text lines using the scissors tool (++c++).
+
+![image: Cutting a text line.](img/segment/scissors.gif "You can cut one or several text lines using the scissors tool (++c++).")
+
+!!! tip
+    All advanced features are described in a help window, just click on **?** in the segmentation panel.
```

Comment:
You put an uppercase for "Segmentation" every other time so you should do it here too